tests/system/robot/chromeTests.py

```diff
@@ -2104,3 +2104,125 @@ def test_focus_mode_on_focusable_read_only_lists():
 		]),
 		message="focus mode - focus list item and turn on focus mode"
 	)
+
+
+def test_ARIASwitchRole():
+	"""
+Ensure that ARIA switch controls have an appropriate role and states in browse mode.

```

Comment:
Add tab to string for consistent indentation

```suggestion
	Ensure that ARIA switch controls have an appropriate role and states in browse mode.
```